tests/python/pants_test/backend/project_info/tasks/test_filedeps.py

```diff
@@ -13,9 +13,10 @@
 from pants.backend.project_info.tasks.filedeps import FileDeps
 from pants.build_graph.register import build_file_aliases as register_core
 from pants_test.task_test_base import ConsoleTaskTestBase
+from pants_test.test_base import TestGenerator
 
 
-class FileDepsTest(ConsoleTaskTestBase):
+class FileDepsTestBase(ConsoleTaskTestBase, TestGenerator):
```

Comment:
Let's not rename this class any more, because it's now an actual test class rather than a base class.